src/Components/TabsItem/TabsItem.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState } from "react";
+import "./TabsItem.scss";
+import Icons from "../Icons/Icons";
+
+function TabsItem({ title, content, icon }) {
+  const [clicked, setClicked] = useState(false);
+
+  const handleContentVisibility = () => {
+    setClicked(!clicked);
+  };
+
+  return (
+    <>
+      <div onClick={handleContentVisibility} className="tabs">
```

Comment:
`tabs` is not proper naming here. It only contains the tab title. Maybe `tab-title` instead?

Reply:
Done.

Reply:
Done.